qutebrowser/mainwindow/tabwidget.py

```diff
@@ -478,6 +480,7 @@ def minimumTabSizeHint(self, index, ellipsis: bool = True) -> QSize:
             index: The index of the tab to get a size hint for.
             ellipsis: Whether to use ellipsis to calculate width
                      instead of the tab's text.
+                     Forced to false for pinned tabs.
```

Comment:
`false` -> `False` - and while you're at it, mind indenting this and the previous line a space more? :wink: